endpoint_handlers_test.go

```diff
@@ -689,6 +689,42 @@ func TestEndpointCreateBadRequest(t *testing.T) {
 	templates.BadRequestTest(t, rec)
 }
 
+// TestEndpointCreateInvalidTenant tests bad request is returned
+// when create request contains source id that is not owned by tenant
+func TestEndpointCreateInvalidTenant(t *testing.T) {
```

Comment:
I guess the name should be `TestEndpointCreateInvalidSource` then, right?

Reply:
🤔 you're right, that is better name for it 😁